pubspec.yaml

```diff
@@ -44,9 +44,12 @@ dependencies:
   flutter_svg: ^2.0.10+1
   # Logging
   logger: ^2.4.0
-  # Custom Packages
-  rest_client_kit:
-    path: ./packages/rest_client_kit
+  # Mappable
```

Comment:
`# Mappable` is incorrect, we should call it `# Serialization & Deserialization`